libraries/angular/src/advanced-tests.js

```diff
@@ -78,10 +78,10 @@ describe("advanced support", function() {
       let root = fixture.debugElement.nativeElement;
       let wc = root.querySelector("#wc");
       let handled = root.querySelector("#lowercase");
-      expect(handled.textContent).toEqual("false");
+      expect(handled.textContent).to.equal("false");
```

Comment:
is it possible to use `eql` throughout, instead of `equal`?

Reply:
In this case, I think `equal` is the correct method to use since it's not an object. Check out the chai.expect API http://chaijs.com/api/bdd/#method_eql. 
```
@param { Mixed } val
@param { String } msg _optional_
.equal(val[, msg])
```
vs 
```
@param { Mixed } obj
@param { String } msg _optional_
.eql(obj[, msg])
```
Let me know what you think 👍

Reply:
I did some testing and I think `eql` may be what you want because it lets you loosely compare objects and it works with other types.

Reply:
I think GitHub recently changed their behavior where pressing shift-enter now submits the comment instead of doing a line break. Annoying...

Reply:
Works for me!

Reply:
I did some testing:

```js
expect({foo: 'bar'}).to.eql({foo: 'bar'}); // passes
expect({foo: 'bar'}).to.equal({foo: 'bar'}); // fails, because they're different instances
```

```js
expect(42).to.eql(parseInt("42", 10)); // also passes
```